src/borg/remote.py

```diff
@@ -1005,16 +1004,10 @@ def handle_remote_line(line):
                     # of local progress displays.
                     sys.stderr.write('Remote: ' + msg['message'] + '\r')
     elif line.startswith('$LOG '):
-        # This format is used by Borg since 1.1.0b1.
+        # This format is used by borg serve 0.xx, 1.0.x and 1.1.0b1..b5.
         # It prefixed log lines with $LOG as a marker, followed by the log level
         # and optionally a logger name, then "Remote:" as a separator followed by the original
         # message.
-        #
-        # It is the oldest format supported by these servers, so it was important to make
-        # it readable with older (1.0.x) clients.
-        #
-        # TODO: Remove this block (so it'll be handled by the "else:" below) with a Borg 1.1 RC.
```

Comment:
not possible, we still want to support a borg 1.1 client talking to a 1.0 server.